src/models/room.ts

```diff
@@ -1193,8 +1206,17 @@ export class Room extends ReadReceipt<EmittedEvents, RoomEventHandlerMap> {
      * @returns The notification count, or undefined if there is no count
      *          for this type.
      */
-    public getThreadUnreadNotificationCount(threadId: string, type = NotificationCountType.Total): number | undefined {
-        return this.threadNotifications.get(threadId)?.[type];
+    public getThreadUnreadNotificationCount(threadId: string, type = NotificationCountType.Total): number {
```

Comment:
this looks like a breaking change - the function is not flagged as experimental, it's public, and it has a behavioural change.

Reply:
It should have been marked as experimental. And this is only behind a labs flag.
I believe this should not be marked as a breaking change.

Reply:
The js-sdk doesn't have a concept of labs flags though, and we can't guarantee that people *aren't* using it. There's other breaking changes in the SDK already landed anyways, so I think we can just apply the label here and move on.